gradle.properties

```diff
@@ -10,7 +10,7 @@ awsSdkVersion=1.11.273
 azureSdkVersion=1.20.1
 eventBusVersion=2.0.7.RELEASE
 slf4jApiVersion=1.7.12
-guavaVersion=21.0
+guavaVersion=27.1-jre
```

Comment:
I'm curious: why change this?

I asked because I ran into some issues with transitive dependencies in my experiments in cloudbreak, and was curious if y'all would run into the too. I managed to figure out the gradle to exclude stuff if you're interested.

Reply:
because grpc depends on version "26-android" which does not contain some class from normal guava lib, so cloudbreak will not compile. If I update to 27 which is higher than 26-android, then gradle will put v27 on classpath.

Reply:
Yeah,  this is what I ran into. I was able to avoid this with appropriate exclusions. I'll share what  I did and you can decide what to make of it.